kernel: bump 5.10 to 5.10.175

graysky2 opened the pull request:
Manually rebased:
        backport-5.10/611-v5.12-net-ethernet-mediatek-support-setting-MTU.patch

Removed upstreamed:
        bcm47xx/patches-5.10/170-bgmac-fix-initial-chip-reset-to-support-BCM5358.patch[1]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.175&id=cbf11ff3708ff163387da924f80a47ce7c721e9b

graysky2 committed:
x86: fix deprecated CONFIG_MICROCODE_OLD_INTERACE

We use late loading[1] so need to set this option despite upstream adding a
kernel taint when this option is set.  See discussion in PR#12149 for more details.

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/arch/x86/Kconfig?id=v5.10.173&id2=v5.10.172

1. https://github.com/openwrt/openwrt/blob/master/target/linux/x86/base-files/lib/preinit/02_load_x86_ucode

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 committed:
kernel: tcindex classifier has been retired

https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/net/sched?h=v5.10.173&id=18c3fa7a7fdbb4d21dafc8a7710ae2c1680930f6

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 commented:
Not sure how to fix the error for lantiq

hauke commented on target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch:
```diff
@@ -225,18 +225,16 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
--
+ #define UVC_QUIRK_WAKE_AUTOSUSPEND	0x00002000
 +#define UVC_QUIRK_MOTION		0x00001000
 +#define UVC_QUIRK_SINGLE_ISO		0x00002000
```
Please use new numbers for these new defines.

graysky2 replied:
Done will push soon

hauke commented:
You can just remove `unsigned long physical_memsize = 0L;` from the lantiq patch. 
`physical_memsize` was previously used to set in `mttgpr(7, physical_memsize);` and this was changed to hard code 0, see:
https://github.com/gregkh/linux/commit/bccccd43a06dfea5f5a8295a0cb384d74b90a9b3

graysky2 committed:
kernel: bump 5.10 to 5.10.173

Manually rebased:
        ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch

All other patches automatically rebased.

Signed-off-by: John Audia <therealgraysky@proton.me>

graysky2 committed:
kernel: bump 5.10 to 5.10.174

No patches needed to be rebased.

Signed-off-by: John Audia <therealgraysky@proton.me>

hauke commented on target/linux/x86/config-5.10:
```diff
@@ -218,7 +218,7 @@ CONFIG_MEMFD_CREATE=y
 CONFIG_MICROCODE_AMD=y
 CONFIG_MICROCODE_INTEL=y
-CONFIG_MICROCODE_OLD_INTERFACE=y
+# CONFIG_MICROCODE_LATE_LOADING is not set
```
Please activate it here too.

graysky2 replied:
done

graysky2 committed:
kernel: bump 5.10 to 5.10.175

Manually rebased:
	backport-5.10/611-v5.12-net-ethernet-mediatek-support-setting-MTU.patch

Removed upstreamed:
	bcm47xx/patches-5.10/170-bgmac-fix-initial-chip-reset-to-support-BCM5358.patch[1]

All other patches automatically rebased.

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=v5.10.175&id=cbf11ff3708ff163387da924f80a47ce7c721e9b

Signed-off-by: John Audia <therealgraysky@proton.me>
